### PR TITLE
Add new maintainer

### PR DESCRIPTION
**Pull request by quartox**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by quartox** [comment]
@conda-forge-admin, please rerender

**Comment by quartox** [comment]
@ocefpaf asking to become a maintainer for this feedstock. I am also creating a new version release but not sure how to have someone review my work since this is a data package.

**Comment by ocefpaf** [comment]
> @ocefpaf asking to become a maintainer for this feedstock.

Awesome!

> I am also creating a new version release but not sure how to have someone review my work since this is a data package.

Looks like there are no actual releases, right? In that case we need to keep pointing to a specific commit and bumping the "datever."

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,13 @@
-{%set name = "nltk_data" %}
-{%set version = "2017.10.22" %}
+{% set version = "2017.10.22" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: What is the datever for this commit? Can you update it here?

Comment (quartox) [reply recipe/meta.yaml]: I grabbed the latest commit and updated the detever for that commit. I also updated the hash to fit the hash of that tar file.

Comment (quartox) [reply recipe/meta.yaml]: Only thing I am not sure about is if I need to require the newest version of the `nltk` package to go along with the latest version of the data.

Comment (ocefpaf) [reply recipe/meta.yaml]: > I grabbed the latest commit and updated the detever for that commit. I also updated the hash to fit the hash of that tar file.

Thanks!

> Only thing I am not sure about is if I need to require the newest version of the `nltk` package to go along with the latest version of the data.

Probably not but having that won't hurt users.

**Pull request merged**